src/Plugin/QueueWorker/CronEncryptedFieldUpdate.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * @file
+ * Contains Drupal\field_encrypt\Plugin\QueueWorker\CronEncryptedFieldUpdate.
+ */
+
+namespace Drupal\field_encrypt\Plugin\QueueWorker;
+
+use Drupal\field_encrypt\Plugin\QueueWorker\EncryptedFieldUpdateBase;
+
+/**
+ * A Queue Worker that updates field encryption on cron run.
```

Comment:
Why CRON? What is the use case?

Wouldn't this potentially lag encryption updates? What happens if the data is processed _before_ update?